### PR TITLE
Ruby Warnings

### PR DESCRIPTION
**Pull request by amatsuda**
Here are some fixes for ruby warnings that I see in my app.

### CODE REVIEW

**Comment by estolfo** [comment]
Hi @amatsuda
Thank you for this. Why have you included the csasl.bundle file?

**Comment by amatsuda** [comment]
@estolfo 😱 I'm so sorry! That was just a mistake. Fixed and `push-f`ed.

**Comment by estolfo** [comment]
@amatsuda no worries, thank you for this PR!

File: lib/mongo/cluster.rb [hunk 1]
```diff
@@ -154,6 +154,7 @@ def initialize(seeds, monitoring, options = Options::Redacted.new)
       @update_lock = Mutex.new
       @pool_lock = Mutex.new
       @topology = Topology.initial(seeds, monitoring, options)
+      @cursor_reaper = nil
```
Comment (estolfo) [inline lib/mongo/cluster.rb]: The @cursor_reaper is initialized on line 175 though..

**Pull request merged**